entries/p/planfix.com.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Planfix": {
+    "domain": "planfix.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://planfix.com/help/Enabling_two-factor_authentication_for_employees",
```

Comment:
Maybe it would be better to use [the group](https://planfix.com/help/Two-factor_authentication) instead of a page?

Reply:
I think you are right! ❤️